bedrock/products/templates/products/vpn/includes/macros.html

```diff
@@ -200,3 +200,24 @@ <h2 class="vpn-content-media-heading">{{ heading }}</h2>
     </a>
   </div>
 {%- endmacro %}
+
+{% macro vpn_resource_center_cta(referral_id, link_text, alt_link_text, class_name) -%}
+  <div class="vpn-nav-cta">
+    {% if vpn_available %}
+      {{ vpn_product_referral_link(
+        referral_id=referral_id,
+        page_anchor='#pricing',
+        link_text=link_text,
+        class_name=class_name,
+        optional_attributes= {
+          'data-cta-text' : 'Get Mozilla VPN',
+          'data-cta-type' : 'button',
+        }
+      ) }}
+    {% else %}
```

Comment:
Please add some extra logic to hide the button from `VPN_EXCLUDED_COUNTRY_CODES` countries 🥺 🙏

Reply:
Yeah, okay, it's out of scope for this work.